### PR TITLE
CLC-5154 - View-As: save/un-save user automatically shows less

### PR DESCRIPTION
**Pull request by edwlook**
https://jira.ets.berkeley.edu/jira/browse/CLC-5154

### CODE REVIEW

File: src/assets/javascripts/angular/controllers/widgets/studentLookupController.js [hunk 1]
```diff
@@ -218,24 +218,55 @@
...
+        } else {
+          tab.users = [];
+        }
+        for (var j = 0; j < newTab.users.length; j++) {
+          tab.users.push(newTab.users[j]);
+        }
+      }
     };
+
+    $scope.admin.tabs = [
```
Comment (christianv) [inline src/assets/javascripts/angular/controllers/widgets/studentLookupController.js]: @edwlook this is almost a copy/paste with the `updateTabs`, is there no way to combine them? Make it a bit more DRY?

File: src/assets/javascripts/angular/controllers/widgets/studentLookupController.js [hunk 2]
```diff
@@ -218,24 +218,55 @@
...
         },
-        { // Recent tab
-          name: 'Recent',
+        {
           error: $scope.admin.recentUsersError,
           users: $scope.admin.storedUsers.recent
         }
       ];
+
+      for (var i = 0; i < newTabs.length; i++) {
```
Comment (christianv) [inline src/assets/javascripts/angular/controllers/widgets/studentLookupController.js]: @edwlook can we add some comment(s) to explain this for loop a bit more?

**Comment by edwlook** [comment]
@christianv Addressed your comments, let me know if you have any questions.

**Comment by christianv** [comment]
Lgtm

**Pull request merged**